cirq-core/cirq/circuits/circuit.py

```diff
@@ -1831,6 +1843,41 @@ def _load_contents_with_earliest_strategy(self, contents: 'cirq.OP_TREE'):
     def __copy__(self) -> 'cirq.Circuit':
         return self.copy()
 
+    def freeze(self) -> 'cirq.FrozenCircuit':
+        """Gets a frozen version of this circuit.
+
+        Repeated calls to `.freeze()` will return the same FrozenCircuit
+        instance as long as this circuit is not mutated.
+        """
+        from cirq.circuits import FrozenCircuit
```

Comment:
nit: should this be a full import path `cirq.circuits.frozen_circuit`?

Reply:
This was copied from what we had in `AbstractCircuit.freeze` previously. I don't feel strongly either way, though. Will change to use full import path.